store/storetest/channel_store.go

```diff
@@ -576,12 +576,12 @@ func testChannelStoreGetByName(t *testing.T, ss store.Store) {
 		t.Fatal("Missing id should have failed")
 	}
 
-	if r1 := <-ss.Channel().GetByName(o1.TeamId, o1.Name, false); r1.Err != nil {
```

Comment:
I guess `r1` comes to express "result 1", if we only have `r1` and we are reusing it, probably makes more sense to use directly `result`.

Reply:
Fixed